Custom-Audio-Driver/Custom-Audio-Driver/OTDefaultAudioDevice-Mac.m

```diff
@@ -565,16 +565,12 @@ static void update_playout_delay(OTDefaultAudioDeviceMac* device) {
     device->_playoutDelayMeasurementCounter++;
         
     if (device->_playoutDelayMeasurementCounter >= 100) {
-            uint32_t tempPlayoutDelay = 0;
-            tempPlayoutDelay += (int)(device->_playout_AudioUnitProperty_Latency * 1000000);
+            device->_playoutDelay = 0;
+            device->_playoutDelay += (int)(device->_playout_AudioUnitProperty_Latency * 1000000);
```

Comment:
This is not the same as only:
`device->_playoutDelay = (int)(device->_playout_AudioUnitProperty_Latency * 1000000);`